ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/zoomToHome.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import SplitButton from '.'
+import styled from '../../styles/styled-components'
+import { hot } from 'react-hot-loader'
+
+type voidFunc = () => void
+
+type Props = {
+  goHome: voidFunc
```

Comment:
✏️ Even though there's no difference I would replace `voidFunc` with `() => void`. Mainly because I haven't seen anywhere else in the codebase where we declare a type like `voidFunc`. 

I would also compare declaring a type like `voidFunc` to creating a constant to represent the empty string in java.

Reply:
A void function isn't empty, it's just a function that doesn't return anything.